gems/activerecord/6.0/activerecord-generated.rbs

```diff
@@ -19731,16 +19731,16 @@ module ActiveRecord
     def empty?: () -> untyped
 
     # Returns true if there are no records.
-    def none?: () -> untyped
+    def none?: () ?{ (untyped) -> bool } -> bool
```

Comment:
```suggestion
    def none?: () ?{ (untyped) -> boolish } -> bool
```

Could you change the returned value of the block to `boolish`? `boolish` is an alias of `top`, and `bool` is an alias of `true | false`.
In this case, the block can return any values other than `bool`, such as `nil`. So it should be `boolish` instead of `bool`.

We need to update the other methods in the same way also.

ref: https://github.com/ruby/rbs/blob/b16ea15a11cf9cdc098da94080268e4ec0767955/docs/syntax.md#bool-or-boolish

Reply:
Thanks for your review!

I changed the return value of the block to `boolish`.
Thank you for your kind explanation.